man/rmd-fragments/navbar-configuration.Rmd

```diff
@@ -13,7 +13,7 @@ navbar:
 
 It makes use of the the six built-in components:
 
--   `intro`: "Get Started", which links to a vignette with the same name as the package.
+-   `intro`: "Get Started", which links to a vignette with the same name as the package. Note that dots (`.`) in the package name need to be replaced by hyphens (`-`) in the vignette filename to be recognized as the intro.
```

Comment:
Could you please make this a footnote? It's not going to affect that many packages.

Reply:
Done